src/actionTypes.js

```diff
@@ -344,6 +344,9 @@ type EventUpdateMessageAction = {|
   type: typeof EVENT_UPDATE_MESSAGE,
   edit_timestamp: number,
   message_id: number,
+  message_ids: number[],
+  stream_id: number,
```

Comment:
This should be optional, according to the docs. (It's there only when the message was sent to a stream.)